Add configurable keybindings for workspace selection in expo plugin

kontura opened the pull request.

soreau commented on plugins/single_plugins/expo.cpp:
```diff
@@ -146,6 +209,11 @@ class wayfire_expo : public wf::plugin_interface_t
 
+        for (size_t i = 0; i < keyboard_select_cbs.size(); i++)
+        {
+            output->add_activator(keyboard_select_options[i], &keyboard_select_cbs[i]);
```
This is a nice idea but to be frank, I don't care for this solution. It means you have to add a binding per workspace and that can get ridiculous quick. Instead, I think it might be better if there were prev/next workspace bindings or some binding with arrows to move around the workspace selection. Also click-to-select-workspace could be implemented, maybe double-click to select and end expo and/or you could have scroll wheel bound to prev/next.

ammen99 replied:
This is a more general issue which I am going to resolve in a next version of our config library. This approach is fine until then.

ammen99 commented:
Sorry for delaying this for so long, can you rebase on top of master?
Also please add more bindings (I'd say up to 16) until we come up with a more generic solution in `wf-config`

kontura commented:
I have rebased, hopefully I got the new wf-config API right.
Also I have added more bindings but only up to 10, because I am not sure what kind of default keys do you want.

ammen99 commented:
@kontura Yeah it looks right, maybe just add one more comment to the default config file so that users can figure out that more bindings work as well (no need to set defaults or something).

ammen99 commented:
Just tested, seems to work fine
Thanks!
